docker/php/Dockerfile

```diff
@@ -1,10 +1,73 @@
 FROM php:7.2-fpm-stretch
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-  libpq-dev libcurl4-openssl-dev libpng-dev zlib1g-dev && \
-  pecl install apcu_bc-1.0.4 && \
-  docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql && \
-  docker-php-ext-install pdo pdo_pgsql pgsql && \
-  docker-php-ext-enable apcu && \
-  docker-php-ext-install curl iconv gd opcache && \
-  rm -r /var/lib/apt/lists/* /tmp/*
+ARG log_errors
+ARG display_errors
+ARG site_name
+ARG app_env
+ARG app_secret
+ARG trusted_proxies
+ARG trusted_hosts
+ARG no_reply_address
+ARG mailer_url
+ARG database_url
+ARG aws_ssm_name_db_url
+ARG aws_ssm_region
+
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y libpq-dev libcurl4-openssl-dev libpng-dev zlib1g-dev
+RUN pecl install apcu_bc-1.0.4
```

Comment:
All the apt-related stuff should be a single `RUN` command, just like in the other file